File: src/main/java/com/procurement/submission/model/dto/response/BidsUpdateStatusDetailsResponseDto.java
package com.procurement.submission.model.dto.response;

import com.fasterxml.jackson.annotation.JsonCreator;
import com.fasterxml.jackson.annotation.JsonProperty;
import lombok.Getter;

@Getter
public class BidsUpdateStatusDetailsResponseDto {

    private BidUpdateDto bid;

    @JsonCreator
    public BidsUpdateStatusDetailsResponseDto(@JsonProperty("bid") final BidUpdateDto bid) {
        this.bid = bid;
    }
}

File: src/main/kotlin/com/procurement/submission/model/dto/response/BidsUpdateStatusDetailsResponseDto.kt
package com.procurement.submission.model.dto.response

import com.fasterxml.jackson.annotation.JsonCreator
import com.fasterxml.jackson.annotation.JsonInclude


@JsonInclude(JsonInclude.Include.NON_NULL)
data class BidsUpdateStatusDetailsResponseDto @JsonCreator constructor(

        val bid: BidUpdateDto
)
